### PR TITLE
Index page design updated

### PR DESCRIPTION
**Pull request by DilwoarH**
- Adds additional column for status
- make index page layout changes (filters now 1/4, results now 3/4)
- changes font size to 16 for table row text

https://trello.com/c/jok0QIR6/286-index-page-design

## Screenshot

screen shot 2018-11-13 at 13 44 13

https://trello.com/c/cJmGlDh9/374-update-design-for-index-page-in-content-publisher

### CODE REVIEW

File: app/assets/stylesheets/objects/_grid.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .app-grid-column--end {
   text-align: right;
 }
+
+.govuk-width-container {
+  max-width: 1200px;
```
Comment (alex-ju) [inline app/assets/stylesheets/objects/_grid.scss]: As discussed, if we're going to go with a new breaking point we should do it by defining it in `media-queries`. The rest of the code is fine, but we should discuss this with the rest of the team and understand implications.

File: app/assets/stylesheets/views/index.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.results-table-wrapper {
+  .govuk-table__body {
```
Comment (alex-ju) [inline app/assets/stylesheets/views/index.scss]: I would move these to `app/assets/stylesheets/utilities/_overwrites.scss` as it's an actual overwrite.

File: app/views/documents/index.html.erb [hunk 3]
```diff
@@ -24,7 +24,9 @@
         <%= govspeak_to_html t("documents.index.search_results.guidance") %>
       </div>
     <% else %>
-      <%= render "documents/index/results" %>
+      <div class="results-table-wrapper">
```
Comment (alex-ju) [inline app/views/documents/index.html.erb]: We won't need the additional wrapper as the overwrite should be applied at the project level.

File: app/views/documents/index/_results.html.erb [hunk 4]
```diff
@@ -28,13 +29,12 @@
               },
               metadata: {
                 document_type: document.document_type_schema.label,
-                document_create: t("documents.index.search_results.creator", creator: document.creator&.name || I18n.t!("documents.index.search_results.unknown_user")),
```
Comment (alex-ju) [inline app/views/documents/index/_results.html.erb]: We've lost the `document_create` on the way or we dropped it intentionally?

Comment (DilwoarH) [reply app/views/documents/index/_results.html.erb]: Hey alex - we dropped it intentionally. @soniaturcotte said we don't need both creator and last editor here.

Comment (alex-ju) [reply app/views/documents/index/_results.html.erb]: OK

**Pull request merged**